src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -30,51 +30,54 @@
          (i18n/label :t/bridge)
          (i18n/label :t/send))]
       [quo/summary-tag
-       {:token        (if collectible? "" token-display-name)
-        :label        (str amount " " token-display-name)
-        :type         (if collectible? :collectible :token)
-        :image-source (if collectible? image-url :eth)}]]
+       (cond-> {:token (if collectible? "" token-display-name)
+                :label (str amount " " token-display-name)
+                :type  (if collectible? :collectible :token)}
+         collectible? (assoc :image-source image-url))]]
```

Comment:
Previously we had `:image-source` receiving `:eth` which is not a valid image source, now is fixed so when we bridge the  token image is properly displayed